app/scripts/metamask-controller.js

```diff
@@ -216,9 +228,9 @@ module.exports = class MetamaskController extends EventEmitter {
       version,
     })
 
-    // tx mgmt
```

Comment:
Another removed comment. I think I agree that these comments are not useful, but not sure that we should remove them in this PR.